modules/core/src/main/java/org/apache/synapse/core/axis2/ProxyService.java

```diff
@@ -317,7 +318,7 @@ public AxisService buildAxisService(SynapseConfiguration synCfg, AxisConfigurati
         if (synapseEnv != null) {
             synapseEnvironment = (SynapseEnvironment) synapseEnv.getValue();
         }
-        auditInfo("Building Axis service for Proxy service : " + name);
+        auditInfo("Building Axis service for Proxy service");
```

Comment:
Shall we leave the messages as they were? I wander if this might invalidate any alerting queries made in the existing logs.

Reply:
I too was thinking , but anyway as we are doing this in a new release and not as an update .  can't we mention in the docs or sthng. Else this will be a redundant info , wdyt.

Reply:
I'm ok with both, just thinking about the migrating users life :)

Reply:
Changed it back to the way it :)